scIB/integration.py

```diff
@@ -283,6 +287,18 @@ def runBBKNN(adata, batch, hvg=None):
         corrected = bbknn.bbknn(adata, batch_key=batch, neighbors_within_batch=25, copy=True)
     return corrected
 
+
+def runSaucie(adata, batch):
+    saucie = SAUCIE.SAUCIE(adata.X.shape[1])
+    loader = SAUCIE.Loader(adata.X.todense(), labels=adata.obs[batch])
+    saucie.train(loader, steps=1000)
+    adata.obsm['X_emb'] = saucie.get_embedding(loader)[0]
+    adata.X = saucie.get_reconstruction(loader)[0]
```

Comment:
Is this just reconstructed input data, or batch-removed feature data? If the latter, it's fine. But if it's only a reconstruction, we shouldn't include this as it's not the right thing to test.

Reply:
It's the batch corrected feature data